apps/web/client/src/app/project/[id]/_components/right-panel/chat-tab/chat-messages/message-content/tool-call-simple.tsx

```
@@ -77,149 +74,141 @@ export function ToolCallSimple({
+    return str.length > maxLength ? str.substring(0, maxLength) + '...' : str;
+}
+
+const getLabel = (toolName: string, toolPart: ToolUIPart) => {
+    try {
+        switch (toolName) {
+            case TERMINAL_COMMAND_TOOL_NAME:
+                return 'Terminal';
+            case LIST_BRANCHES_TOOL_NAME:
+                return 'Listing branches';
+            case SEARCH_REPLACE_EDIT_FILE_TOOL_NAME:
+                const params = toolPart.input as z.infer<typeof SEARCH_REPLACE_EDIT_FILE_TOOL_PARAMETERS>;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)